3rdparty/python/lockfiles/user_reqs.txt

```diff
@@ -16,20 +16,20 @@
 #     "freezegun==1.1.0",
 #     "humbug==0.2.7",
 #     "ijson==3.1.4",
-#     "packaging==21.0",
-#     "pex==2.1.61",
-#     "psutil==5.8.0",
+#     "packaging==21.3",
+#     "pex==2.1.65",
+#     "psutil==5.9.0",
 #     "pytest<6.3,>=6.2.4",
 #     "requests[security]>=2.25.1",
 #     "setproctitle==1.2.2",
 #     "setuptools<58.0,>=56.0.0",
 #     "toml==0.10.2",
-#     "types-PyYAML==5.4.3",
-#     "types-freezegun==0.1.4",
-#     "types-requests==2.25.0",
-#     "types-setuptools==57.0.0",
-#     "types-toml==0.1.3",
-#     "typing-extensions==3.10.0.2"
+#     "types-PyYAML==6.0.3",
+#     "types-freezegun==1.1.6",
+#     "types-requests==2.27.5",
+#     "types-setuptools==57.4.7",
```

Comment:
Aha - these 2 bumps (freezegun & setuptools) trigger the unused type[ignore] warnings. I'll hand roll those fixes into those non-production files instead of altering locks.